_posts/en/meetings/2016-04-28-meeting.md

```diff
@@ -0,0 +1,100 @@
+---
+title: IRC meeting summary for 2016-04-28
+permalink: /en/meetings/2016/04/28/
+name: 2016-04-28-meeting
+type: meetings
+layout: page
+lang: en
+version: 1
+---
+{% include _toc.html %}
+
+- [Link to this week logs](http://bitcoinstats.com/irc/bitcoin-core-dev/logs/2016/04/28#l1461877211.0)
```

Comment:
I'm noticing that bitcoinstats.com is regularly down and missing messages. Please can you switch to the botbot.me URL? https://botbot.me/freenode/bitcoin-core-dev/ or link to the meetingbot dump linked in the minutes summary http://www.erisian.com.au/meetbot/bitcoin-core-dev/2016/bitcoin-core-dev.2016-04-28-19.00.log.html